src/creds/creds.c

```diff
@@ -45,6 +45,8 @@ static int arg_newline = -1;
 static sd_id128_t arg_with_key = _CRED_AUTO;
 static const char *arg_tpm2_device = NULL;
 static uint32_t arg_tpm2_pcr_mask = UINT32_MAX;
+static uint32_t arg_tpm2_literal_mask = UINT32_MAX;
+static uint8_t arg_tpm2_pcr_literal[24][SHA256_DIGEST_SIZE];
```

Comment:
as mentioned elsewhere. Please just make this a `Hashmap` object, keyed by a combination of pcr number and algorithm